### PR TITLE
Cleanup storage RendererRD

### PR DESCRIPTION
**Pull request by BastiaanOlij**
Just a few small changes I ran into while going through the setup of the mobile renderer. 

@akien-mga I kept this in 3 commits because they are 3 separate changes but let me know if you want me to squash it.

### CODE REVIEW

File: servers/rendering/renderer_rd/renderer_scene_gi_rd.h [hunk 1]
```diff
@@ -653,8 +653,7 @@ class RendererSceneGIRD {
 	~RendererSceneGIRD();
 
 	// !BAS! Can we merge these two inits? Possibly, need to check
```
Comment (akien-mga) [inline servers/rendering/renderer_rd/renderer_scene_gi_rd.h:655]: Obsolete comment.

**Comment by BastiaanOlij** [comment]
Found another one that is just silly to have doubled in a subclass so changed this back to draft

File: servers/rendering/renderer_rd/renderer_scene_gi_rd.h (hunk 1 above)
Comment (BastiaanOlij) [reply servers/rendering/renderer_rd/renderer_scene_gi_rd.h:655]: Cheers!

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**